openstudiocore/src/model/Model.hpp

```diff
@@ -331,7 +341,10 @@ class MODEL_API Model : public openstudio::Workspace {
   //@}
 
   /** Load Model from file. */
-  static boost::optional<Model> load(const path& p);
+  static boost::optional<Model> load(const path& osmPath);
+
+  /** Load Model and WorkflowJSON from files. */
+  static boost::optional<Model> load(const path& osmPath, const path& workflowJSONPath);
```

Comment:
Why can't we just pass workflowJSONPath and figure out osmPath from that?

I believe we may have discussed this and the reason might be to avoid parsing the osw or something, but please remind me. Seems to complicate things with both arguments.